compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -2055,6 +2055,28 @@ void OMR::X86::CodeGenerator::doBinaryEncoding()
             }
          }
 
+      if (estimateCursor->getNext() == NULL && estimateCursor->getKind() != TR::Instruction::IsPadding)
```

Comment:
Why isn't this done outside the loop through all instructions rather than checked on each iteration (i.e., instruction)?  This operation almost literally applies to the very end of the instruction stream and it would seem it could be done last.  Unless I'm misinterpreting...